### PR TITLE
Please add new file

### PR DESCRIPTION
**Pull request by code-later**
Super amazing!
# hello

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
     <img src="taco.gif" />
 
-    <p>Hello World</p>
+    <p>Hello World!!1</p>
```
Comment (code-later) [inline index.html:20]: Was soll denn das?

File: help.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Help</title>
+    <meta robots="nofollow,index">
```
Comment (code-later) [inline help.html:5]: Rly?

File: index.html (hunk 1 above)
Comment (timomeh) [inline index.html:20]: lot-of-feelings

**Comment by code-later** [comment]
:shipit:

**Pull request merged**